assets/js/app.js

```diff
@@ -0,0 +1,4 @@
+import 'bootstrap';
+import Geocoding from './modules/Geocoding';
```

Comment:
Bin mir nicht sicher, aber ich glaube die `import SOMETHING from LIB` Inhalte landen nicht in der finalen Datei.

Reply:
ggf. brauchst du nur folgendes schreiben, damit es klappt:
```
export {Geocoding, Map, Search}
```